src/runtime-rs/crates/shimv2/src/logger.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::os::unix::fs::OpenOptionsExt;
+
+use anyhow::{Context, Result};
+
+use crate::Error;
+
+pub(crate) fn set_logger(path: &str, sid: &str, is_debug: bool) -> Result<slog_async::AsyncGuard> {
```

Comment:
very basic question: what would be a `sid` in this context?

Reply:
The sid is pod sandbox id.